ansible/roles/artemis/templates/broker.xml.j2

```diff
@@ -31,7 +31,9 @@ under the License.
 
         <!-- this could be ASYNCIO or NIO
          -->
-        <journal-type>ASYNCIO</journal-type>
+        <journal-type>MAPPED</journal-type>
```

Comment:
For your use case you have fsync turned off, when this is the case MAPPED is more performant, the default is fsync turned on, in this case ASYNCIO is more performant. 

The majority of orgs that use JMS brokers have the requirement it must be sync'd to disk for before ack, thus fsync is turned on by default.